src/app.css

```diff
@@ -1,8 +1,5 @@
-
```

Comment:
Double check that this has no negative effects when looking at a page that has a sticky nav and scrolls.